content/blog/entries/why-lektor/contents.lr

```diff
@@ -0,0 +1,76 @@
+title: Why Lektor?
+---
+categories:
+outreachy
+tech
+open-source
+---
+author: dhruvi16
+---
+pub_date: 2020-05-13
+---
+body:
+
+
+<div style="text-align: center;">
+  <figure>
+    <img src="cover.jpeg"/>
+    <figcaption>
+      <em>
+        why?why?why?
+      </em>
+    </figcaption>
+  </figure>
+</div>
+
+I wrote this article while prepping for my Outreachy internship. I was trying to find out why Lektor is being used for the project and found out really convincing reasons for the same.
+
+Let’s start with basic types of website -
+
+Static website: Here the webpages are pre-loaded to the server and when the request is made, these pages are served to the client as it is. These webpages can use HTML, CSS, and Javascript. By static, it does not mean that these webpages are devoid of user interactivity. These pages can be highly reactive (because of course JS) but yes they are not generated on the server and are static in that manner.
+
+<div style="text-align: center;">
+  <figure>
+    <img src="static.jpg"/>
+    <figcaption>
+      <em>
+        Static Website.
+      </em>
+    </figcaption>
+  </figure>
+</div>
+
+Dynamic website: Here the webpages are generated by the server. It is connected to a database and all the data is fetched from there. It uses server-side scripting and also client-side scripting (if necessary). It is called dynamic because the webpages are supposed by the server.
+
+<div style="text-align: center;">
+  <figure>
+    <img src="dynamic.jpg"/>
+    <figcaption>
+      <em>
+        Dynamic website.
+      </em>
+    </figcaption>
+  </figure>
+</div>
+
+So why go static? — Because the vast, vast majority of websites will be read many more times than they will be updated. This is crucial because dynamic content does not come for free. It needs server resources and because program code is running there it needs to be kept up to date for to ensure there are no security problems that are left unpatched. Also when a website gets a sudden spike of traffic a static website will stay up for longer on the same server than a dynamic one that needs to execute code.
+
+There are certain drawbacks of static websites we are going to focus on here and how lektor helps us to combat those.
```

Comment:
```suggestion
There are certain drawbacks of static websites we are going to focus on here and how Lektor helps us to combat those.
```
Punctuation is important. Small things like proper capitalisation go a long way.